### ISSUE TITLE
Release final 1.1 version

### ISSUE THREAD

**Issue by astrofrog** [astropy/astropy#3519]
**NOTE** For now these are just major issues ported over from 1.0 that were not completed - we should review and edit these at a later time:

**Integration of units:**

* [ ] Implement masked quantities (#1852) - @adrn @mhvk 

**Coordinates, Time, and Observatories:**

* [ ] automate `IERS` updates (#1697) and other data with expiration dates. @mhvk @taldcroft @eteq
* [ ] barycentric coordinates and velocities, ideally at VLBI precision (if things go quickly enough)

**Stabilize modeling API:**

* [ ] implement support for Quantities(#1464) @nden @embray @wkerzendorf
* [ ] add 'default_fitter', 'last_fitter' and 'fit' method to models (#1330, #979) @nden @embray @wkerzendorf
* [ ] add support for getting uncertainties on parameters (#1827)

**Miscellaneous:**

* [ ] Implement any backward-incompatible API changes needed for the future generalized WCS (even if the full thing isn't implemented) - @nden @perrygreenfield @jehturner
* [ ] Remove things scheduled for removal, in config, units and utils.  (#3500).
* [x] :warning: We need to update the contributor list! :warning: 

In addition, the following documentation/help-related issues should be addressed:

* [ ] Implement a help system tied to exceptions/warnings - @demetri

Lower priority issues

* [ ] Better integration of FITS tables and VO tables with the Table class - @mdboom @taldcroft @embray

**Comment by astrofrog** [comment]
:warning: We need to update the contributor list! :warning:

**Comment by astrofrog** [comment]
Individual commits that should be backported:

* f23a545bb08b20cf8fa70a9adb41ba9e79e7d79e

**Comment by astrofrog** [comment]
Need to update the number for issues/pull requests/contributors at the last minute

**Comment by embray** [comment]
I think maybe this issue should be renamed to "1.2" and moved to that milestone, with a new issue similar to it created for 1.1.  That or make one for 1.2 and move a bunch of the above checkboxes over to it.  

It's just funny because almost none of the checkboxed items in this list are going to be done in the v1.1 release, since other priorities came up.

**Comment by mhvk** [comment]
A new issue is better I'd think...
Before starting that, it would be quite helpful to discuss missing pieces in astropy, and perhaps assign some priority to them. But perhaps it is best done first on a wiki page?

**Comment by eteq** [comment]
@mhvk @embray -   how about we close this one when the final release is done (as we've done for past releases), and then move these to a new issue for v1.2.  We can then use those as a starting point for  a developer hangout to discuss priorities.

**Comment by embray** [comment]
@eteq Unfortunate, but sounds good.

**Comment by mhvk** [comment]
Yes, makes most sense.  And we did get quite a bit done for 1.1, so I'm not going to feel too bad.  Though we also need to think about how to get a few more contributors...

**Comment by eteq** [comment]
Actually, I don't know that contributor growth is a problem per se, @mhvk (see the plot below)... But we *do* need to get some of them more involved in "deep-dive" issues, I think.

![contributors](https://cloud.githubusercontent.com/assets/346587/11601945/b80691b6-9aa4-11e5-91ab-5e853d6c0433.png)

**Comment by mhvk** [comment]
That's very encouraging indeed! Possibly we can get a few more by asking specifically for "seconds" and "thirds" for the different modules (definitely needed for `units`...)

**Comment by embray** [comment]
That's total *cumulative* # of contributors so of course it always goes up.  Would like to see # of new contributors per <period>, and also # of contributors with, say, > 50 commits.

**Comment by eteq** [comment]
Yes, it definitely has to go up or stay flat, but the point I'm making is that it *is* continuing to go up.

The other two are interesting questions, though - the code I used to create the plot above is at https://github.com/astropy/astropy-tools/blob/master/astropy_status_plots.py (the `commiters_plot` function), so anyone is welcome to try teasing out more analytics like that...

**Comment by astrofrog** [comment]
I think it'd also be interesting to make a plot of the number of contributors in a moving window of say 6 months (but I also don't have time to try that out!)

**Comment by embray** [comment]
Incidentally, my sadly neglected "blog" has a post about using Pandas and `git log` to generate statistics about git repos, if anyone wants to play with this: http://iguananaut.net/blog/programming/git-logs-in-pandas.html

**Comment by eteq** [comment]
Release done! https://pypi.python.org/pypi/astropy/1.1

**Comment by taldcroft** [comment]
:fireworks:

**Comment by embray** [comment]
¯\_(ツ)_/¯

**Comment by bsipocz** [comment]
:tada: for 1.1
